src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -97,12 +98,22 @@ CorProfiler::Initialize(IUnknown* cor_profiler_info_unknown) {
   }
 
   // get path to integration definition JSON files
-  const WSTRING integrations_paths =
+  WSTRING integrations_paths =
       GetEnvironmentValue(environment::integrations_path);
 
   if (integrations_paths.empty()) {
-    Warn("Profiler disabled: ", environment::integrations_path,
-         " environment variable not set.");
+    auto profiler_home_path =
+        GetEnvironmentValue(environment::profiler_home_path);
+
+    if (!profiler_home_path.empty()) {
+      const auto fallback_integration_path = std::filesystem::path(profiler_home_path) / "integrations.json";
```

Comment:
We can't use `std::filesystem` unless we update to a newer build system on Linux.

Reply:
I started working on updating the Linux build a few months ago. It's on branch `lpimentel/update-linux-build`.

Reply:
For this PR, which solution would you prefer:
1) Writing our own routine that does simple path appending (you can see that work in some of the recent commits that I reverted)
2) Updating the Linux build to take advantage of `std::filesystem`?

Reply:
Trying to do #2 right now, starting with the changes from your branch. I'll let you know how that goes

Reply:
I've spent all day trying to update the build system just to use the `filesystem::append` method but the result is that `std::experimental::filesystem` is available but not `std::filesystem`. I'm going to give up on this approach for the PR because I spent an entire day on this already. Unless you can help me take care of that, I will roll our own simple path function for this PR.